### PR TITLE
Add Alby

### PR DESCRIPTION
**Pull request by secondl1ght**
I have added Alby as a tutorial connection option for RaspiBlitz users. Please let me know if you require any further information, thanks!

### CODE REVIEW

**Comment by rootzoll** [comment]
Marked for v1.8 release .. will take a detailed look after Bitcoin2022 conference.

File: README.md [hunk 1]
```diff
@@ -171,6 +171,7 @@ You can connect the following Wallet-Apps to your RaspiBlitz:
 * **Fully Noded** (iOS) [details](https://apps.apple.com/us/app/fully-noded/id1436425586)
 * **SendMany** (Android) [details](https://github.com/fusion44/sendmany/blob/master/README.md)
 * **Sphinx Chat App** (Android & iOS) [details](https://sphinx.chat)
+*	**Alby** (Desktop) [details](https://getalby.com)
```
Comment (bumi) [inline README.md]:
```suggestion
* **Alby** (Desktop) [details](https://getalby.com)
```

**Review by bumi — COMMENTED** [review]
looks good. 
but is there a way to easily test this locally?

**Comment by bumi** [comment]
@rootzoll I just actually ran that code (by c&p the code) on my raspiblitz and I think I need to make some updates to this. working on it.

**Comment by secondl1ght** [comment]
thanks @bumi, I didn't have a raspiblitz to test on so it may need some adjustments.

**Comment by rootzoll** [comment]
You can use this running ticket to report on progress on the Alby final testing: #3011

**Pull request merged**